pkgs/by-name/vc/vcpkg/package.nix

```
@@ -32,12 +37,21 @@ stdenvNoCC.mkDerivation (finalAttrs: {
   installPhase = ''
       runHook preInstall
 
-      mkdir -p "$out/bin" "$out/share/vcpkg/scripts/buildsystems"
-      cp --preserve=mode -r ./{docs,ports,triplets,scripts,.vcpkg-root,versions,LICENSE.txt} "$out/share/vcpkg/"
+      mkdir -p "$out/bin" "$out/libexec/vcpkg" "$out/share/vcpkg/scripts/buildsystems"
+      cp --preserve=mode -r ./{.git,docs,ports,triplets,scripts,.vcpkg-root,versions,LICENSE.txt} "$out/share/vcpkg/"
 
       ${lib.optionalString doWrap ''
+        # If you use a builtin-baseline in a vcpkg.json file, then vcpkg will
+        # try to run “git show” on $out/share/vcpkg. By default, Git will fail
+        # if a Git repo isn’t owned by the current user. We create a wrapper
+        # for git here to make sure that the “git show” command succeeds, even
+        # if the user running vcpkg doesn’t own $out/share/vcpkg.
+        makeWrapper "${git}/bin/git" "$out/libexec/vcpkg/git" \
+          --add-flags -c --add-flags safe.director="$out/share/vcpkg"
```

In any case, there is a small typo here


```suggestion
          --add-flags -c --add-flags safe.directory="$out/share/vcpkg"
```

---

Fixed.